### PR TITLE
SSPROD-2053: Added Anchore Reporting k8s YAMLs

### PR DESCRIPTION
**Pull request by yathi-sysdig**
This PR adds k8s YAMLs required for deploying the anchore reporting service.

### CODE REVIEW

File: sysdigcloud/anchore-reports-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
```
Comment (mbreitung) [inline sysdigcloud/anchore-reports-deployment.yaml]: please add limit and requests

Comment (yathi-sysdig) [reply sysdigcloud/anchore-reports-deployment.yaml]: Done

File: sysdigcloud/anchore-reports-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                  fieldPath: metadata.name
+            - name: ANCHORE_POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: ANCHORE_POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+          # See comments in sysdigcloud/config.yaml on how to specify anchore.https.proxy:
```
Comment (markyjackson-taulia) [inline sysdigcloud/anchore-reports-deployment.yaml]: This seems kind of wrong but my bigger question is this also referenced in the larger documentation and just not in the ConfigMap?

Comment (yathi-sysdig) [reply sysdigcloud/anchore-reports-deployment.yaml]: @markyjackson-taulia are you asking if this process is documented someplace? https://sysdig.atlassian.net/wiki/spaces/security/pages/769425745/OnPrem+-+How+to+configure+a+proxy+for+the+Anchore+engine

**Pull request merged**